### PR TITLE
Changing Uberon release location, again (ready to merge)

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: config/uberon.yml [hunk 1]
```diff
@@ -67,13 +70,93 @@ entries:
...
+    - from: /releases/2021-02-12/
+      to: https://raw.githubusercontent.com/obophenotype/uberon/v2021-02-12/
     - from: /releases/2021-02-12/uberon.obo
       to: https://raw.githubusercontent.com/obophenotype/uberon/v2021-02-12/uberon.obo
 
+- prefix: /releases/
+  replacement: https://github.com/monarch-initiative/mondo/releases/download/v
+  tests:  
+    - from: /releases/2022-03-31/uberon.obo
+      to: https://github.com/monarch-initiative/mondo/releases/download/v2022-03-31/uberon.obo
```
Comment (jamesaoverton) [inline config/uberon.yml:158]: I spot-checked some of these, and I got a 404 for this one.

File: config/uberon.yml [hunk 2]
```diff
@@ -20,28 +20,31 @@ entries:
...
-  
+
 - exact: /composite-metazoan.obo
   replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-metazoan.obo
 
 - exact: /experimental/ceph.owl
   replacement: http://purl.obolibrary.org/obo/ceph.owl
 
+- prefix: /subsets/
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/v
```
Comment (jamesaoverton) [inline config/uberon.yml:40]: This one worries me a little, but I trust you know best.

Comment (matentzn) [reply config/uberon.yml]: iiii the v was wrong, thanks for catching that!

File: config/uberon.yml (hunk 1 above)
Comment (matentzn) [reply config/uberon.yml:158]: Yes, it does not exist yet, but it will at some point soon :) Thanks!

**Pull request merged**